### DISCUSSION TITLE
Possible to publish new release of 2.8.x?

### DISCUSSION THREAD

**Discussion by Samyssmile** [spring-projects/spring-kafka#2418] [Q&A] [answered]
In the last few days some commits are done into 2.8.X Branch. This changes are very important for us. Its possble to make new release, so we can receive it over maven repository?

If not, what's the release cycle here?

**Comment by garyrussell** [comment, marked as answer]
?? There is only one commit since 2.8.9, and that is a new feature.

The next release date is currently October 17.
